Scripts/application.py

```diff
@@ -0,0 +1,58 @@
+import sqlite3
+import json
+from typing import Dict
+from flask import Flask
+app = Flask(__name__)
+
+@app.route('/')
+def hello_world():
+    return 'Hello World!'
+    
+@app.route('/api/v1/person')
+def return_people():
+    # returns a json of all people
+    with sqlite3.connect('../db/allYourSantaAreBelongToUs.db') as connection:
+        cursor = connection.cursor()
+        #get all of the people
+        cursor.execute('SELECT * FROM people')

```

Comment:
SELECT * is a little unpredictable, for example if you add a new column it could potentially cause a bug. Can you select only the fields you're interested in?

Reply:
Thanks for the tip.